artifacts/pom.xml

```diff
@@ -119,10 +119,10 @@
 		<authentication-wrapper.fileName>authentication-wrapper.jar</authentication-wrapper.fileName>
 		<!-- esignet mock authentication wrapper -->
 		<esignet-wrapper.location>/usr/share/nginx/html/artifactory/libs-release-local/esignet/esignet-wrapper</esignet-wrapper.location>
-		<esignet-mock-wrapper.version>0.9.0</esignet-mock-wrapper.version>
+		<esignet-mock-wrapper.version>0.9.1</esignet-mock-wrapper.version>
 		<esignet-mock-wrapper.fileName>esignet-mock-wrapper.jar</esignet-mock-wrapper.fileName>
 		<!-- esignet IDA wrapper -->
-		<esignet-ida-wrapper.version>1.2.0.1-B3</esignet-ida-wrapper.version>
+		<esignet-ida-wrapper.version>1.2.1-SNAPSHOT</esignet-ida-wrapper.version>
```

Comment:
this the release 1.2.0.1 branch, why we are using 1.2.1-SNAPSHOT ?

Reply:
corrected the version number to 1.2.0.1-B4